ingeniamotion/configuration.py

```diff
@@ -161,6 +161,35 @@ def default_brake(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS) ->
         """
         self.disable_brake_override(servo, axis)
 
+    def check_configuration(
+        self, config_path: str, axis: Optional[int] = None, servo: str = DEFAULT_SERVO
+    ) -> None:
```

Comment:
I think It would be best if the method return a boolean.

Reply:
The issue specified that the ingenialink function should throw an exception when the check fails:
![image](https://github.com/ingeniamc/ingeniamotion/assets/64021569/28991b75-8596-4e5b-b457-9870125352d8)
I could change this function to catch the error and return a boolean, but I'm not sure if this is what we need.